packages/zoe/src/zoeService/zoeStorageManager.js

```diff
@@ -219,7 +223,30 @@ export const makeZoeStorageManager = (
             keyword,
             assetKind,
             displayInfo,
-            reason => E(state.adminNode).terminateWithFailure(reason),
+            // TODO this terminates (or suspends) only the client zcf vat
+            // that made a ZCFMint that therefore asked the zoeService to
+            // `makeZoeMint`. Since the potential bad state is at that moment
+            // only in the zoeService vat, it is fine to terminate
+            // the zcf vat asynchronously, but before it receives and further
+            // messages from the zoeService vat.
+            //
+            // However, since this ZoeMint may still be
+            // available, we must also terminate/suspend this zoeService
```

Comment:
```suggestion
            // available, we must also panic this zoeService
```

Reply:
Before vat suspension, definitely not. We must suspend the chain.

Even with vat suspension, I don't see how. Ideas?

Reply:
Resolving